pkg/machines/libvirt-common.js

```diff
@@ -185,6 +197,83 @@ export function getSingleOptionalElem(parent, name) {
     return subElems.length > 0 ? subElems[0] : undefined; // optional
 }
 
+export function parseNetDumpxml(netXml) {
+    const netElem = getNetworkElem(netXml);
+    if (!netElem) {
+        return;
+    }
+
+    const forwardElem = netElem.getElementsByTagName("forward")[0];
+    const bridgeElem = netElem.getElementsByTagName("bridge")[0];
+    const ipElems = netElem.getElementsByTagName("ip");
+    const mtuElem = netElem.getElementsByTagName("mtu")[0];
+
+    const ip = parseNetDumpxmlForIp(ipElems);
+    const device = bridgeElem ? bridgeElem.getAttribute("name") : undefined;
+    const mtu = mtuElem ? mtuElem.getAttribute("size") : undefined;
+    let mode = forwardElem ? forwardElem.getAttribute("mode") : "none";
+    if (!mode)
+        mode = "nat"; // if mode is not specified, "nat" is assumed, see https://libvirt.org/formatnetwork.html#elementsConnect
+
+    return {
```

Comment:
There is not need to put them in new line since it's going to be less than 80 chars if inline.